### PR TITLE
Add emoji in label to project boards

### LINKED ISSUE #13880: Project boards: Emojis in Labels missing
- Gitea version (or commit ref): 1.13.0
- Git version: 2.14.3
- Operating system: Ubuntu Server 18.04.5
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [x] Yes (provide example URL) https://try.gitea.io/modmew8/LabelEmojiTest/projects/400
  - [ ] No

## Description

Currently emojis in labels on project boards are not being replaced:

grafik

In comparison to the issue list:

grafik

## The fix

I'm currently unable to deploy and create a pull request myself, but this one line in `templates/repo/projects/view.tmpl` would need a replacement (line 163 on master): (replacing `{{.Name}}` with `{{.Name | RenderEmoji}}`)

```
<a class="ui label has-emoji" href="{{$.RepoLink}}/issues?labels={{.ID}}" style="color: {{.ForegroundColor}}; background-color: {{.Color}}; margin-bottom: 3px;" title="{{.Description}}">{{.Name}}</a>
```
to
```
<a class="ui label has-emoji" href="{{$.RepoLink}}/issues?labels={{.ID}}" style="color: {{.ForegroundColor}}; background-color: {{.Color}}; margin-bottom: 3px;" title="{{.Description}}">{{.Name | RenderEmoji}}</a>
```

Maybe someone can try out and validate, I just tested it by creating `custom/templates/repo/projects/view.tmpl` and replacing this single line, resulting in the correct behaviour:

grafik

### PR DESCRIPTION
**Pull request by rakshith-ravi**
As title.

Fixes #13880

CC @modmew8

### CODE REVIEW

File: templates/repo/projects/view.tmpl [hunk 1]
```diff
@@ -160,7 +160,7 @@
 						</div>
 						<div class="extra content">
 							{{ range .Labels }}
-							<a class="ui label has-emoji" href="{{$.RepoLink}}/issues?labels={{.ID}}" style="color: {{.ForegroundColor}}; background-color: {{.Color}}; margin-bottom: 3px;" title="{{.Description}}">{{.Name}}</a>
+							<a class="ui label has-emoji" href="{{$.RepoLink}}/issues?labels={{.ID}}" style="color: {{.ForegroundColor}}; background-color: {{.Color}}; margin-bottom: 3px;" title="{{.Description}}">{{.Name | RenderEmoji}}</a>
```
Comment (zeripath) [inline templates/repo/projects/view.tmpl]: So .has-emoji was removed from master a while ago - not sure how this was missed in this code-base. Also it appears we should be RenderEmojiPlaining the description too.

```suggestion
							<a class="ui label" href="{{$.RepoLink}}/issues?labels={{.ID}}" style="color: {{.ForegroundColor}}; background-color: {{.Color}}; margin-bottom: 3px;" title="{{.Description | RenderEmojiPlain}}">{{.Name | RenderEmoji}}</a>
```

**Review by zeripath — APPROVED** [review]

**Comment by rakshith-ravi** [comment]
@zeripath should we add this to 1.13.1?

**Comment by zeripath** [comment]
Yup - I was just checking what the code base looked like on 1.13 - it appears that this can be backported as it stands.

**Review by 6543 — APPROVED** [review]

**Comment by 6543** [comment]
@rakshith-ravi pleace send backport :+1:

**Comment by rakshith-ravi** [comment]
@6543 how do I do that?

**Comment by 6543** [comment]
checkout to the release branch & cherry pick 729f0f5

should I do it for you?

**Comment by rakshith-ravi** [comment]
Nah I'll do it. I wanna learn to backport :D

**Comment by 6543** [comment]
@rakshith-ravi some hints:

```sh
git fetch --all
git checkout -f upstream/release/v1.13 -b backport_13978
git cherry-pick 729f0f5f7b
git push -u fork backport_13978
```
-> create pull

fork is remote to your fork
upstream is remote to official repo

**Comment by 6543** [comment]
@rakshith-ravi did a backport for you -> #14021

**Comment by rakshith-ravi** [comment]
Hey. Thanks. My laptop just broke and I wouldn't have been able to do it till the weekend. Sorry about the delay

**Pull request merged**